components/Shared/BlurredBgImage.tsx

```diff
@@ -0,0 +1,37 @@
+import { BlurredBgImageStyled } from "./BlurredBgImage.styled";
+import React from "react";
+
+interface Props {
+  bgColor?: string;
+  bgImageUrl: string;
+  height?: string;
+  width?: string;
+}
+
+const BlurredBgImage: React.FC<Props> = ({
+  bgColor = "transparent",
+  bgImageUrl,
+  height,
+  width,
+  ...restProps
+}) => {
+  return (
+    <div
+      style={{
+        backgroundColor: bgColor,
+      }}
+      data-testid="blurred-bg-image"
+      {...restProps}
+    >
+      <BlurredBgImageStyled
+        css={{
+          backgroundImage: `url(${bgImageUrl})`,
```

Comment:
Since we are blurring, we could get away sending a smaller thumbnail. Not necessary but saves some data.
https://dcapi.rdc-staging.library.northwestern.edu/api/v2/works/6c2e4d68-c55e-4f3b-91ad-ae37bb63ba39/thumbnail?size=20

```suggestion
          backgroundImage: `url(${bgImageUrl}?size=20)`,
```

Reply:
Sure.  I didn't even know there were query params possible on this endpoint.